functions/New-DbaSsisCatalog.ps1

```diff
@@ -0,0 +1,117 @@
+﻿Function New-DbaSsisCatalog {
+<#
+.SYNOPSIS 
+Enables the SSIS Catalog on a SQL Server 2012 +
+
+.DESCRIPTION
+After installing the SQL Server Engine and SSIS you still have to enable the SSIS Catalog. This function will enable the catalog and gives the option of supplying the password. 
+
+.PARAMETER SqlServer
+SQL Server you wish to run the function on.
+
+.PARAMETER SqlCredential
+Credenitals used to connect to the SQL Server
+
+.PARAMETER SsisCredential
+Optional Password that will be used for the security key in SSISDB
+
+.NOTES 
+Author: Chrissy LeMaire (@cl), netnerds.net
+Requires: sysadmin access on SQL Servers
+
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/New-DbaSsisCatalog
+
+.EXAMPLE   
+New-DbaSsisCatalog -SqlServer DEV01 -SsisCredential MySuperPassword
+
+Creates the SSIS Catalog on server DEV01
+
+#>
+	[CmdletBinding(SupportsShouldProcess = $true)]
+	Param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlInstance")]
+		[string[]]$SqlServer,
+        [System.Management.Automation.PSCredential]$SqlCredential,
+        [string]$SsisPassword
```

Comment:
Oh, never plain text 🦈 Make this a secure string, or can use the credential object to make it easier for folks.